rfc/networked-dot-storage.md

```diff
@@ -0,0 +1,174 @@
+
+Eventually we may need some kind of regional caching.
+
+### Repair service for both W3UP and Filecoin
+
+Repair service means a service that creates permaneant reliable storage from a series of time-limited storage deals with providers who might fail.
+
+It's a vital missing component today on Filecoin AND a missing component of W3S as soon as their is more than one instance. We've already encountered this problem indirectly as we've thought about deal renewals and the reality that deal terms on Filecoin do not coincide in time with users joining and leaving W3UP.
+
+It needs to exist for Filecoin AND it needs to exist for W3S.
+
+*** But isn't this a point of centralization if everyone uses it? ***
+
+Absolutely. It's important that key coordination functions in the network like a repair service eventually operate through distributed consensus and produce verifiable results.
+
+### Shared w3filecoin pipeline
+
+W3Filecoin is an excellent service for aggregating data on to Filecoin. At the same time, running it is NOT easy. While the basic aggregation functions are simple, making Filecoin deals is not. Multiple instances of W3UP should be able to share an instance of W3Filecoin. 
+
+*** But isn't this a point of centralization if everyone uses it? ***
+
+Absolutely. It's important that key coordination functions in the network like w3-filecoin eventually operate through distributed consensus and produce verifiable results.
+
+### Verification Mechanisms For Retrievals from W3UP Instances
+
+See [Proofs Overview](https://w3sat.notion.site/Filecoin-SPs-in-the-W3S-Network-16a9b66e7ef3425bbee390c6481bd752?pvs=4)
+
+It is important that the W3UP service level agreements are not compromised across the network by poorly performing or malicious W3Up instances.
+
+### Filecoin.Storage
+
+Produce an implementation of W3UP and W3Filecoin that can be run by a Filecoin L1 Storage Provider. In order words, a Filecoin storage provider should be able to be more than a backup on our network -- and eventually get seperate compensation -- if they can meet our service level requirements. See [Filecoin SPs in the W3S Network](https://w3sat.notion.site/Filecoin-SPs-in-the-W3S-Network-16a9b66e7ef3425bbee390c6481bd752?pvs=4)
+
+## FAQ
+
+### Is decentralization the wrong word to use?
+
+Decentralization is both a concept so overloaded it lacks any coherence, and a narrative that our product team has validated is indeed important to many market segments in web3. Actual customers have told us they won't use our product unless it's "decentralized". Likely, in some cases this is a novel kind of CYA (cover-your-ass) -- where enterprise customers want SOC-2 compliance in a data center so they can blame someone else if data gets lost, web3 customers want services they use to be "decentralized" so they can't get called out by the internet for not being "web3". Decentralization is both an annoying meme and an important narrative we have to contend with.
```

Comment:
> Actual customers have told us they won't use our product unless it's "decentralized".

Right totally, it's just that as you describe they all mean slightly different things, so when they say this we need to dig deeper and figure out which bits they genuinely find important. Our product team coming back from a call with a customer and saying "they won't use us unless we're decentralized" is worse than no feedback at all - it just throws a log on every little fire we each maintain to our preferred definitions of the word and makes these conversations harder and less concrete. I don't think we necessarily need to stop using the word entirely, but the phrase "we need to do X because it makes us more decentralized and that's what our customers want!" is up there with "electrolytes, it's what plants crave!" in the pantheon of misguided sentiments.

Reply:
Fair, and fwiw I don't disagree with any of this - I am mostly just harping on internal use of "decentralization" without more context on what we mean specifically when we're talking about the nuts and bolts of what we want to build and why - totally agree that we can and should be taking advantage of its value as a marketing meme! 

If anything I think we have been too shy about highlighting the parts of the current state of the project that imho _are_ "decentralized" in meaningful ways simply because our backend happens to be implemented in terms of a well known cloud service provider rather than being "hosted on the decentralized web" somehow - open specs, open software, (nearly) trivial deployment of third party w3up instances are all things we have right now that address what I think are the interesting user-facing facets of "decentralization" and I think we should lean into that.

> part of the enthusiasm for decentralization is simply an ill defined impulse for a better kind of service on the internet

hard agree, this is why I'm here for sure! and the reality of this is part of why I get so worked up by imprecise usage of this term when we talk about what we want to build - I think there's a lot of real value here that tends to get overshadowed by the marketing/vibes-y usages of it, and I _really_ want to be building things that address this ill-defined desire rather than something that just uses the term to get pageviews - I think I'm in good company on that front!

Reply:
Can we just declare ourselves decentralized ? That way customers can tell us what they actually need or tell us how what we are doing is not decentralized enough which is effectively telling us what they need.